modules/setting/lfs.go

```diff
@@ -66,6 +67,11 @@ func loadLFSFrom(rootCfg ConfigProvider) error {
 		LFSClient.BatchSize = 20
 	}
 
+	if LFSClient.BatchOperationConcurrency < 1 {
+		// match the default git-lfs's `lfs.concurrenttransfers`
+		LFSClient.BatchOperationConcurrency = 3
```

Comment:
the default in lfs client is actually 8, not 3: https://github.com/git-lfs/git-lfs/blob/main/lfshttp/client.go#L89 , but I've documented what you set here: https://gitea.com/gitea/docs/pulls/88

Reply:
Good catch .... actually there are different documents .... the default value also ever changed. 😅 

The new document: https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-config.adoc also says 8

Reply:
I will update it to 8 and update the documents.

Thank you very much.